src/renderer/src/layouts/default/index.tsx

```diff
@@ -1,10 +1,16 @@
 import { ReactNode } from 'react'
 import { LanguageSwitcher } from '../../components/LanguageSwitch'
 
-export const DefaultLayout = ({ children }: { children: ReactNode }) => {
+export const DefaultLayout = ({
+  children,
+  themeVarient = 'light',
+}: {
+  children: ReactNode
+  themeVarient?: 'light' | 'dark'
```

Comment:
Lets stay away from a light and dark mode right now. We don't even have the finalized colors yet....

Reply:
sorry this is bad naming on my part. This just determines the foreground colour of the language select since there are some screens where it will be over a dark background and some on a light background.

Reply:
> lets scope this pr a little more and take out the fonts
How do you suggest i do that? A commit removing them will do?

> I also don't think it is necessary to add all those fonts
Yeah probably not, but a lot of these are fallback fonts, I guess we can be confident about font file support in an electron app right? So perhaps we can select one font file type and use that. Perhaps we don't need all the weights either, perhaps Sabella can answer that one.

Reply:
yeah, just use a commit to take them out!